src/shared/ui/survey/StackedCheckboxItem/StackedCheckboxItem.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC } from 'react';
+import { StyleSheet } from 'react-native';
+
+import StackedCheckboxConfig from './types';
+import { CheckBox } from '../..';
+import { StackedItemsGrid } from '../StackedItemsGrid';
+import { StackedRowItemValue } from '../StackedItemsGrid/types';
+
+type Props = {
+  value: Array<string[]>;
+  onChange: (value: Array<string[]>) => unknown;
+  config: StackedCheckboxConfig;
+};
+
+const StackedCheckboxItem: FC<Props> = ({ value = [], onChange, config }) => {
+  const onValueChange = (option: StackedRowItemValue, itemIndex: number) => {
+    if (!value[itemIndex]?.length) {
+      value[itemIndex] = [];
+    }
+
+    if (value[itemIndex].includes(option.name.en)) {
+      value[itemIndex] = value[itemIndex].filter(
+        valueItem => valueItem !== option.name.en,
```

Comment:
I think this has to be name 
In refactored backend there will be name : string instead of name: { en: string }.

Reply:
Fixed